docs/developing-contracts/state.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_label: State functions
+sidebar_position: 3
+---
+
+# Store data
+
+Persistent data of the Gear contact is stored in the same way as in a classic program and does not require initialization of the external storage.
+
+```rust
+// describe state structure
+pub struct State {
+    votes_received: BTreeMap<String, i32>,
+}
+
+// initialize state itsalfe
+static mut STATE: State = State::new();
+```
+
+If you program in Rust or other object-oriented languages, then you should be familiar with most types. However `ActorId` type shoud be new in Gear contract development.
+
+:::info
+`ActorId` is a special type 
+:::
```

Comment:
Strange remark for me

Reply:
Probably, it needs some more details to be more informative
E.g. it's 32 bytes array defining id on a blockchain